dnf/util.py

```diff
@@ -60,7 +60,11 @@ def ensure_dir(dname):
         if not os.path.isdir(dname):
             raise IOError("%s is not a directory" % dname)
     else:
-        os.makedirs(dname, mode=0o755)
+        try:
+            os.makedirs(dname, mode=0o755)
+        except OSError as e:
+            if e.errno != os.errno.EEXIST:
```

Comment:
Could you rather remove the lines above the new code and add just ```or not os.path.isdir(dname)``` into the new ```if```?